lib/plug/parsers.ex

```diff
@@ -126,6 +126,13 @@ defmodule Plug.Parsers do
       plug Plug.Parsers,
            parsers: [:urlencoded, {:json, json_decoder: Jason}]
 
+  It is also possible to pass the `:json_decoder` as an MFA, useful for passing
+  options to the JSON decoder:
```

Comment:
I always try to avoid acronyms and "cryptic" terms when writing docs. I know we call this "MFA" above in the docs, but I still suggest to change this to:

```suggestion
  It is also possible to pass the `:json_decoder` as a `{module, function, args}` (MFA) tuple,
  useful for passing options to the JSON decoder:
```